packages/rum-core/src/domain/view/setupViewTest.specHelper.ts

```diff
@@ -1,14 +1,22 @@
-import type { BuildContext } from '../../../test'
+import { Observable } from '@datadog/browser-core'
+import { mockRumConfiguration, setupLocationObserver } from '../../../test'
+import type { LifeCycle } from '../lifeCycle'
 import { LifeCycleEventType } from '../lifeCycle'
 import type { ViewEvent, ViewOptions } from './trackViews'
 import { trackViews } from './trackViews'
 
 export type ViewTest = ReturnType<typeof setupViewTest>
 
-export function setupViewTest(
-  { lifeCycle, location, domMutationObservable, configuration, locationChangeObservable }: BuildContext,
-  initialViewOptions?: ViewOptions
-) {
+interface ViewTrackingContext {
+  lifeCycle: LifeCycle
+  initalLocation?: string
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  initialLocation?: string
```